demos-europe/demosplan-core
# bug (refs T27614): Refactor user form fields

**Ticket:** https://yaits.demos-deutschland.de/T27614

Input fields for user form fields had different sizes. 
- DpUserFormFields now uses DpInput for all inputs 
- Use DpSelect 
- Use tailwaind classes 
- Fix spacing between inputs and button row


### How to review/test
Login as Support. Go to Nutzende -> Neuen Nutzer anlegen. Fields should have the same size and work as expected.
